db/vita_partners.yml

```diff
@@ -484,3 +484,24 @@ vita_partners:
   source_parameters:
     - uwsuncoast
   logo_path: 'partner-logos/UWSC.png'
+- name: Temple University
+  zendesk_instance_domain: eitc
+  zendesk_group_id: '360012561073'
+  display_name: Temple University
+  source_parameters:
+  - tuvs
+  logo_path: 'partner-logos/temple.jpg'
+- name: Goodwill of the Olympics and Rainier Region
+  zendesk_instance_domain: eitc
+  zendesk_group_id: '360012588394'
+  display_name: Goodwill of the Olympics and Rainier Region
+  source_parameters:
+  - orrr
+  logo_path: 'partner-logos/goodwill_orr.png'
```

Comment:
@GAV07 It looks like this image path does not match the file name

Reply:
Yes, I forgot to add the extra 'r' to the link